tests/bootstrap.php

```diff
@@ -14,6 +14,31 @@
 // @see https://core.trac.wordpress.org/browser/trunk/tests/phpunit/includes/functions.php
 require_once $_tests_dir . '/includes/functions.php';
 
+/**
+ * Force plugins defined in a constant (supplied by phpunit.xml) to be active at runtime.
+ *
+ * @filter site_option_active_sitewide_plugins
+ * @filter option_active_plugins
+ *
+ * @param array $active_plugins
+ * @return array
+ */
+function xwp_filter_active_plugins_for_phpunit( $active_plugins ) {
```

Comment:
@kidunot89 what was the reason to name this function `xwp_filter_...` instead of `stream_filter_...`?

Reply:
@kopepasah It was copied from another plugin.